### ISSUE TITLE
Add an end-to-end example to Readme

### ISSUE THREAD

**Issue by danielkhan** [jaredhanson/passport#534]
I guess simply adding `var LocalStrategy = require('passport-local').Strategy;` to the sample would lower the barrier for new users drastically.

**Comment by 19smitgr** [comment]
I've just started learning about Passport, and the main thing that confuses me is about the sessions and why serialize user only has to be called once in the entire program. At this point, I assume that calling serializeUser() does something that serializes every request.

**Comment by codypearce** [comment]
I agree that would help, but the readme does already link to 4 tutorials with 4 examples, as well as the links to each strategies' repo. The official docs are also a great resource.

Perhaps the readme could go into a little more detail though.
